src/Runtime/rigidBody.ts

```
@@ -99,6 +99,16 @@ class RigidBodyInner {
     public get hasShadows(): boolean {
         return 0 < this._shadowCount;
     }
+
+    public getShapeReference(): PhysicsShape {
+        return this._shapeReference!;
+    }
+
+    public setShapeReference(shape: PhysicsShape) {
+        this._shapeReference!.removeReference();
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)